.asf.yaml

```diff
@@ -18,6 +18,18 @@ github:
     merge:  false
     rebase: false
 
+  protected_branches:
+    main: {}
+
+  protected_tags:
+    - "releases/*"
+
+  autolink_jira:
+    - SOLR
+
+  collaborators:
+    - solrbot
```

Comment:
> * `collaborators` - list of github user names to allow full PR edit karma

Am curious if that would cover its own PRs only or others' pull requests too, and/or when it (solrbot) would have a need to use full PR edit.

Reply:
The list of users here are allowed full edit rights for all issue/PRs, so it will be users who are not yet committers, but we trust them to do triage. Solrbot would need this right in order to set a label on the PRs it opens. In [`renovate-config.js`](https://github.com/solrbot/renovate-github-action/blob/main/solr/renovate-config.js) we could [configure it to set a `dependencies` label ](https://docs.renovatebot.com/configuration-options/#addlabels) and even labels detailing what kind of dependency it is (e.g. test vs compile). But that is disabled as of now as solrbot does not have that karma.

We could imagine future bots to assign reviewers to PRs intelligently, and such a bot would also be added here. But primarily it would be cool to get 3-5 non-committer developers to volunteer to help with PRs.